pkg/testutils/manifest.go

```diff
@@ -51,6 +51,7 @@ func NewTestManifest(prefix string) *v1beta2.Manifest {
 	}
 }
 
+// GetManifest is only used when manifest still been tracked in kyma.status.
```

Comment:
Sorry, I don't get this comment 😄 It's a test util function that should only be used when? Or what is meant by it?

Reply:
If you check the internal logic of this function, it uses `kyma.status.Modules` to get Manifest CR name, as you know the Manifest CR name contains a hash value, which calculated by kyma and module fqdn which coming from module template, there is not easy way to get this name directly, otherwise we need to parsing moduletemplate, to get it from component descriptor.

However, if the module is disabled, it will not exists in the `kyma.status`, that's the reason why I add this commit, maybe we can enhance the commit a bit, any suggestion?